src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -2223,6 +2229,8 @@ private static void RegisterSerializationProviders(List<TypeInfo> providerTypes)
                 return;
             }
 
+            externalSerializers.Clear();
+            typeToExternalSerializerDictionary.Clear();
```

Comment:
do we actually want to clear everything when registering new serialization providers? shouldn't these be additive?